app/views/companies/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<% provide(:title, "All companies") %>
+<div class="row">
+  <div class="col-md-12 text-center">
+  	<br/><br/><br/>
+    <h4>Subscribed Companies Details</h4>
+    <hr class="star-primary"><br/>
+	<div class="row">
+	  <div class="panel panel-default">
+	    <div class="table-responsive">
+	      <table class="table table-bordered table-hover table-heading">
+	        <thead>
+	          <tr class="success">
```

Comment:
@godfreym  you can put the `text-center` class in the `tr` element and it will affect all the `td` elements